src/cache_manager.cc

```diff
@@ -188,17 +175,12 @@ CacheManager::ParseUrl(const AnyP::Uri &uri)
     Mgr::Command::Pointer cmd = new Mgr::Command();
```

Comment:
I cannot attach this change request to the right line in the code, but the `if` statement above still allows non-magical URLs. That was necessary because `cache_object://...` URLs did not have a magic prefix. This PR should not allow non-magical URLs anymore AFAICT.

You may be tempted to fix this by writing a replacement like this:

```C++
if (!tok.skip(internalMagicPrefix))
        throw TextException("invalid URL path", Here());
```

... but I believe the correct replacement is actually this:

```C++
Assure(tok.skip(internalMagicPrefix));
```

Please double check my logic: Bugs notwithstanding, this method must not be called for non-cache-manager URLs. AFAICT, it is not called for non-cache-manager URLs in the official code. We can and should assert that invariant using Assure().

Reply:
Agreed. Also we should remove the 'define whether the URL is a cache-manager URL' documentation part then.